docs/docs-developers/docs/tutorials/faceid_wallet.md

```diff
@@ -2,11 +2,11 @@
 title: Using FaceID to Sign Transactions
 tags: [local_network, wallet, cli]
 keywords: [wallet, cli wallet, faceid]
-sidebar_position: 5
+sidebar_position: 1
```

Comment:
wdyt about moving this down in the tutorials list? I don't think it should be at the top